src/test/java/HomeWork18.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+public class HomeWork18 extends BaseTest{
+
+    @Test
+
+    public void playSongTest() throws InterruptedException {
+
+        navigateToPage();
+        provideEmail("guadalupe.medina@testpro.io");
+        providePassword("DoingitBig23!");
+        submitButton();
+
+
+    }
+    public void playSong(){
+            WebElement playButton = driver.findElement(By.cssSelector("span[@data-testid='play-btn']"));
+            WebElement playNextSongButton = driver.findElement(By.cssSelector("i[@data-testid='play-next-btn']"));
+
+            playButton.click();
+            playNextSongButton.click();
+
+            Assert.assertTrue(isSongPlaying());
+
+        }
+
+
+        public boolean isSongPlaying() {
+            WebElement soundBarPlay = driver.findElement(By.cssSelector("div[@data-testid ='sound-bar-play']"));
+            return soundBarPlay.isDisplayed();
+
+
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)